examples/transfer-policy-check/src/lib.rs

```diff
@@ -0,0 +1,124 @@
+/*! A contract that showcases how to use policies.
+ *
+ * In particular, it can act as a filter for transfers to an account,
+ * where the filter requires the sender to be from a specific country.
+ *
+ * The contract is initialised with `init`, in which an account address is
+ * provided, which will be set as the state.
+ *
+ * The contract has a single receive entrypoint, `receive`,
+ * which checks whether all sender policies have their country of residence
+ * attribute set to `LOCAL_COUNTRY`. If that is the case, then the `amount`
+ * will be forwarded to the account address held in the state. Otherwise,
+ * the receive function will reject with `ContractError::NotLocalSender`.
+ */
+use concordium_std::*;
```

Comment:
If you want to support `no_std`
```suggestion
#![cfg_attr(not(feature = "std"), no_std)]
use concordium_std::*;
```